### PR TITLE
Added new disposable domains

### PR DESCRIPTION
**Pull request by egorsmkv**
Started from PR number #95

New domains taken from these repos:

- https://github.com/ivolo/disposable-email-domains/
- https://github.com/andreis/disposable

### CODE REVIEW

**Comment by egorsmkv** [comment]
CC @apollonin

**Comment by martenson** [comment]
@ivolo @andreis are you fine with us merging your lists?

**Comment by andreis** [comment]
LGTM

I haven't had time to work on `andreis/disposable` but it needs a complete rewrite. One day...

**Comment by SuN-80** [comment]
@martenson are you gonna merge the lists?

**Comment by martenson** [comment]
@SuN-80 I think so, it is a very large chunk so I did not find the time to skim through it yet.

File: disposable_email_blacklist.conf [hunk 1]
```diff
@@ -818,20 +1229,33 @@ fakedemail.com
...
+fannny.ml
 fansworldwide.de
 fantasymail.de
+farfurmail.tk
+farmerlife.us
+farmtoday.us
 farrse.co.uk
+fartwallet.com
+fassagforpresident.ga
+fast-mail.fr
```
Comment (deguif) [inline disposable_email_blacklist.conf]: This should probably go to `whitelist.conf`.

File: disposable_email_blacklist.conf [hunk 2]
```diff
@@ -818,20 +1229,33 @@ fakedemail.com
...
 fastacura.com
 fastchevy.com
 fastchrysler.com
+fastemails.us
```
Comment (deguif) [inline disposable_email_blacklist.conf]: This should probably go to `whitelist.conf` too.

File: disposable_email_blacklist.conf [hunk 3]
```diff
@@ -1819,168 +2824,307 @@ plexolan.de
...
 porsh.net
 posdz.com
 postacin.com
 postonline.me
 poutineyourface.com
 powered.name
+powerml.racing
 powlearn.com
+poy.kr
+pp.ua
```
Comment (deguif) [inline disposable_email_blacklist.conf]: See https://github.com/martenson/disposable-email-domains/pull/96, should be removed from the list, or add a specific subdomain.

File: disposable_email_blacklist.conf [hunk 4]
```diff
@@ -2100,6 +3317,8 @@ spambox.info
 spambox.irishspringrealty.com
 spambox.org
 spambox.us
+spamcannon.com
```
Comment (deguif) [inline disposable_email_blacklist.conf]: It is already in the `whitelist.conf`.

File: disposable_email_blacklist.conf [hunk 5]
```diff
@@ -2100,6 +3317,8 @@ spambox.info
...
+spamcannon.net
```
Comment (deguif) [inline disposable_email_blacklist.conf]: It is already in the `whitelist.conf` too.

File: disposable_email_blacklist.conf [hunk 6]
```diff
@@ -2122,6 +3341,9 @@ spamfree24.info
 spamfree24.net
 spamfree24.org
 spamgoes.in
+spamgourmet.com
```
Comment (deguif) [inline disposable_email_blacklist.conf]: It is already in the `whitelist.conf`.

File: disposable_email_blacklist.conf [hunk 7]
```diff
@@ -2122,6 +3341,9 @@ spamfree24.info
...
+spamgourmet.net
```
Comment (deguif) [inline disposable_email_blacklist.conf]: It is already in the `whitelist.conf`.

File: disposable_email_blacklist.conf [hunk 8]
```diff
@@ -2122,6 +3341,9 @@ spamfree24.info
...
+spamgourmet.org
```
Comment (deguif) [inline disposable_email_blacklist.conf]: It is already in the `whitelist.conf`.

File: disposable_email_blacklist.conf [hunk 9]
```diff
@@ -2572,62 +4042,98 @@ wuespdj.xyz
...
+xlgaokao.com
+xloveme.top
 xmail.com
 xmaily.com
 xn--9kq967o.com
+xn--b-dga.vn
+xn--d-bga.net
 xost.us
 xoxox.cc
+xoxy.net
```
Comment (deguif) [inline disposable_email_blacklist.conf]: It is already in the `whitelist.conf`.

File: disposable_email_blacklist.conf [hunk 10]
```diff
@@ -2572,62 +4042,98 @@ wuespdj.xyz
...
 yaqp.com
+yaraon.cf
+yaraon.ga
+yaraon.gq
+yaraon.ml
+yaraon.tk
 ycare.de
 ycn.ro
 ye.vc
+yeah.net
```
Comment (deguif) [inline disposable_email_blacklist.conf]: It is already in the `whitelist.conf`.

File: disposable_email_blacklist.conf [hunk 11]
```diff
@@ -272,279 +350,486 @@ anonbox.net
...
 anonymail.dk
 anonymbox.com
+anonymize.com
 anonymized.org
 anonymousness.com
+anotherdomaincyka.tk
 ansibleemail.com
 anthony-junkmail.com
+anthropologycommunity.com
+antichef.com
```
Comment (deguif) [inline disposable_email_blacklist.conf]: Already in `whitelist.conf`.

File: disposable_email_blacklist.conf [hunk 12]
```diff
@@ -272,279 +350,486 @@ anonbox.net
...
+antichef.net
```
Comment (deguif) [inline disposable_email_blacklist.conf]: Already in `whitelist.conf`.

**Comment by deguif** [comment]
@egorsmkv to avoid duplicate entries with `whitelist.conf`, please use this command: `comm -23 new_file.conf whitelist.conf > disposable_email_blacklist.conf` as stated in the README.

**Pull request closed without merging**